README.md

```diff
@@ -33,8 +33,10 @@ our company's [history and SaaS metrics](https://www.parabol.co/blog/tag/friday-
 ### Prerequisites
 
 - [Node](https://nodejs.org/en/download/)
-- [pnpm](https://pnpm.io/)
-- [Docker Compose](https://docs.docker.com/compose/install/)
+- [PostgreSQL 16.0+](https://www.postgresql.org/) (current specific version available in the [build action](./.github/workflows/build.yml)) with [pgvector plugin](https://github.com/pgvector/pgvector)
+- [Redis 7.0+](https://redis.io) (current specific version available in the [build action](./.github/workflows/build.yml))
```

Comment:
-1 While this is required for running the app, we don't want to make the impression it needs to be manually installed to run the app. Maybe split the "Prerequisites" into "Production" and "Development"?